src/select/utils/use-select.ts

```diff
@@ -14,9 +15,8 @@ import { OptionsListProps } from '../../internal/components/options-list';
 import { FilterProps } from '../parts/filter';
 import { ItemProps } from '../parts/item';
 import { usePrevious } from '../../internal/hooks/use-previous';
-import { BaseKeyDetail } from '../../internal/events';
-import { CancelableEventHandler, fireCancelableEvent, NonCancelableEventHandler } from '../../internal/events/index';
-import { containsOrEqual } from '../../internal/utils/dom';
+import { BaseKeyDetail, NonCancelableEventHandler } from '../../internal/events';
+import { CancelableEventHandler, fireNonCancelableEvent } from '../../internal/events/index';
```

Comment:
I wonder why do we have imports from `internal/events/index` separately? Everything could be imported from  `internal/events`